README.md

```diff
@@ -202,6 +202,22 @@ try {
 ```
 Note that in the closure-based example above, exceptions are intercepted automatically.
 
+## Integration with ServiceLens
+
+As of version 2.4.0, the X-Ray SDK for Java is integrated with [CloudWatch ServiceLens](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/ServiceLens.html). This allows you to use a wide range of new observability features which connect your traces, logs, and metrics in one place.
+
+### Trace ID Injection into Logs
+
+You can automatically inject your current Trace ID into logging statements if you use the Log4J or SLF4J logging frontends. To learn more and enable this feature on your instrumented project, see the [developer guide](https://docs.aws.amazon.com/xray/latest/devguide/xray-sdk-java-configuration.html#xray-sdk-java-configuration-logging). 
+
+### Vended Segment-Level Metrics
+
+The X-Ray SDK can now automatically vend metrics that aggregate information about the segments your application sends as a normal CloudWatch metric. To learn more and enable this feature on your instrumented project, see the [developer guide](https://docs.aws.amazon.com/xray/latest/devguide/xray-sdk-java-monitoring.html#xray-sdk-java-monitoring-enable).
```

Comment:
Custom metric would be a more correct term

Reply:
Fixed!